test/smoke_test/bulk_data_requests.sh

```diff
@@ -8,4 +8,6 @@ go run bcda_client.go -host=api:3000 -clientID=$CLIENT_ID -clientSecret=$CLIENT_
 echo "Running Patient"
 go run bcda_client.go -host=api:3000 -clientID=$CLIENT_ID -clientSecret=$CLIENT_SECRET -endpoint=Patient
 echo "Running Coverage"
-go run bcda_client.go -host=api:3000 -clientID=$CLIENT_ID -clientSecret=$CLIENT_SECRET -endpoint=Coverage
\ No newline at end of file
+go run bcda_client.go -host=api:3000 -clientID=$CLIENT_ID -clientSecret=$CLIENT_SECRET -endpoint=Coverage
+echo "Running Patient All"
+go run bcda_client.go -host=api:3000 -clientID=$CLIENT_ID -clientSecret=$CLIENT_SECRET -endpoint=All
```

Comment:
Can we also add this to the Jenkinsfile for smoke tests, so that the default `/Patient` gets tested in AWS?